mathesar_ui/src/component-library/button/Button.scss

```diff
@@ -23,43 +23,48 @@ button.btn {
   /* Appearances */
   &.btn-default {
     background: var(--color-white);
-    border-color: var(--color-gray-medium);
+    border-color: var(--brand-500);
+    color: var(--brand-500);
 
     &:not(:disabled) {
       &:focus {
-        border-color: #39a0f5;
+        border-color: var(--brand-500);
       }
     }
   }
 
   &.btn-primary {
-    background: #39a0f5;
-    border-color: #39a0f5;
-    color: #fff;
+    background: var(--brand-500);
+    border-color: var(--brand-500);
+    color: var(--white);
+    font-weight: 500;
```

Comment:
I would be cautious of using non-standards font weights like this (anything other than 400 or 700), so I'd suggest removing `font-weight: 500;`. Although perhaps you have some rationale that I don't understand here. My thinking is like this... Since we're using system fonts, I think it'll be difficult for us to preview the effect of non-standard font weights across everyone's different environment. In my experience, non-standard font weights can sometimes look a bit awkward, like, they end up being more bold that you think they'll be, or the subpixel rendering or kerning degrades slightly. If we were defining (and shipping) a variable font, I'd be in support of getting fancy with subtle font weights like this. But since we're using system fonts, I think we ought to stick to `font-weight` values of either `normal` (i.e. 400) or `bold` (i.e. 700).

Reply:
Done 52eb3ef